Symfony/CS/DocBlock/Annotation.php

```diff
@@ -37,7 +37,7 @@ class Annotation
     /**
      * Create a new line instance.
      *
-     * @param Line[] $lines
+     * @param array $lines
```

Comment:
@SpacePossum which version of PhpStorm are you using ? the ``Line[]`` notation works for me. I have it in hundred places in my work project